test/LibraryManager.Mocks/LibraryInstallationState.cs

```diff
@@ -31,5 +31,15 @@ public class LibraryInstallationState : ILibraryInstallationState
         /// The path relative to the working directory to copy the files to.
         /// </summary>
         public virtual string DestinationPath { get; set; }
+
```

Comment:
should LibraryId be reimplemented as Name@Version?  Right now the mock can set all 3 of these independently, which could be a little confusing if it ever comes up.

Reply:
Looks like a lot of the current tests set the `LibraryId` directly in the mock. 
Going to leave it as is for now. When we get to removing usage from libraryId, we should hit these test failures and fix them then.